Citation.cff

```diff
@@ -1,9 +1,12 @@
 cff-version: 1.2.0
 message: "If you use this software, please our paper as below."
 authors:
-- family-names: "M. Conlon et al"
+- family-names: "Conlon et al"
   given-names: " "
 title: "VIVO: a system for research discovery"
+journal: "Journal of Open Source Software"
+volume: "4"
+issue: "39"
 version: 1.0.0
 doi: 10.21105/joss.01182
```

Comment:
There are also url and repository-code metadata.

Reply:
The article url is redundant with the DOI.
For the code url we can use either https://vivoweb.org/download/ or https://github.com/vivo-project/VIVO/releases
```suggestion
doi: 10.21105/joss.01182
repository-code: https://vivoweb.org/download/
```